apps/prairielearn/elements/pl-units-input/pl-units-input.py

```diff
@@ -139,9 +139,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
 
         correct_answer_parsed = ureg.Quantity(correct_answer)
 
-        if (correct_answer_parsed is not None) and (
```

Comment:
`ureg.Quantity(None)` will simply error, this condition could never have been true. I'm not sure if we want additional checks before this or not; I'm inclined to say it's not necessary since no one has complained about weird errors?

Reply:
I agree that I think this is fine. If it was a problem, we should have encountered errors before.